server/container_create.go

```diff
@@ -46,29 +47,54 @@ const (
 	defaultSystemdParent  = "system.slice"
 )
 
-func addOCIBindMounts(mountLabel string, containerConfig *pb.ContainerConfig, specgen *generate.Generator) ([]oci.ContainerVolume, error) {
+type orderedMounts []rspec.Mount
+
+// Len returns the number of mounts. Used in sorting.
+func (m orderedMounts) Len() int {
+	return len(m)
+}
+
+// Less returns true if the number of parts (a/b/c would be 3 parts) in the
+// mount indexed by parameter 1 is less than that of the mount indexed by
+// parameter 2. Used in sorting.
+func (m orderedMounts) Less(i, j int) bool {
+	return m.parts(i) < m.parts(j)
+}
+
+// Swap swaps two items in an array of mounts. Used in sorting
+func (m orderedMounts) Swap(i, j int) {
+	m[i], m[j] = m[j], m[i]
+}
+
```

Comment:
Did you mean to uppercase the three functions above?

Reply:
I'd suggest longer function names above.  If you're new to the project, you'd get a lot of hits grepping for "Len".  Maybe mountLength, mountLess, mountSwap?

Reply:
Please take a look at golang documentation https://golang.org/pkg/sort/#Sort

Reply:
Ah, I see you're implementing the sort interface.  I'd still prefer longer names, but you're right, you can't do so.  Thanks for the 411.